v2.6/getting-started/kubernetes/installation/hosted/kubernetes-datastore/index.md

```diff
@@ -130,14 +130,6 @@ kubectl apply -f {{site.url}}/{{page.version}}/getting-started/kubernetes/instal
 >[Click here to view the above yaml directly.](policy-only/1.7/calico.yaml)
 
 
-### 3. Calico policy-only with flannel networking
-
-The [Canal](https://github.com/projectcalico/canal) project provides a way to easily deploy
-Calico with flannel networking.
-
-Refer to the following [Kubernetes self-hosted install guide](https://github.com/projectcalico/canal/blob/master/k8s-install/README.md)
-in the Canal project for details on installing Calico with flannel.
```

Comment:
Line 49 should also be deleted.
https://github.com/projectcalico/calico/pull/1636/files#diff-c6863f4099a95c01ab3ab89b4ad39898R49